x-pack/plugin/src/test/resources/rest-api-spec/test/sql/whitelist.yml

```diff
@@ -0,0 +1,27 @@
+---
+"SQL methods available in Painless":
+    - do:
+        bulk:
+          refresh: true
+          body:
+            - index:
+                _index: test
+                _type:  doc
+                _id:    1
+            - str: test1
+              int: 1
+    - do:
```

Comment:
I think this is a fairly internal detail so we shouldn't have an integration test for it. I think the normal SQL integration tests cover this use case plenty so we don't *need* this anyway.